src/app/admin/product/all-products.component.ts

```diff
@@ -16,41 +16,12 @@ import {AbstractSearchComponent} from '../../shared/components/abstract-search/a
 })
 export class AllProductsComponent extends AbstractSearchComponent implements OnInit, OnDestroy {
 
-  // tempUrl = '';
-  // noPic;
+  products = [];
 
   ngOnInit() {
     this.key = 'Product';
     super.ngOnInit();
   }
-
```

Comment:
this should not be in this review. please use different local branches to avoid irrelevant diff